src/_cffi_src/openssl/crypto.py

```diff
@@ -94,7 +94,7 @@
 # define OPENSSL_PLATFORM        SSLEAY_PLATFORM
 # define OPENSSL_DIR             SSLEAY_DIR
 #endif
-#if CRYPTOGRAPHY_OPENSSL_LESS_THAN_110
+#if CRYPTOGRAPHY_OPENSSL_LESS_THAN_110 || CRYPTOGRAPHY_IS_LIBRESSL
```

Comment:
`CRYPTOGRAPHY_OPENSSL_LESS_THAN_110` already does the `CRYPTOGRAPHY_IS_LIBRESSL` check so we can revert this.

Reply:
Awesome, thanks for your help!